### PR TITLE
Update operator manifest for v0.3.0

### PR DESCRIPTION
**Pull request by ranyodh**

### CODE REVIEW

**Review by nwneisen — APPROVED** [review]
Is this all generated code? Can we create a ticket to summarize what is done here so that it can be added to the release CI

**Comment by ranyodh** [comment]
@nwneisen 

>Is this all generated code?

Yes, there is a `make` target to generate this in `boundless-operator` repo.

> Can we create a ticket to summarize what is done here so that it can be added to the release CI

Created this ticket: https://mirantis.jira.com/browse/BOP-318

File: deploy/static/boundless-operator.yaml [hunk 1]
```diff
@@ -777,75 +1069,88 @@ spec:
...
+          capabilities:
+            drop:
+            - ALL
+      - args:
+        - --health-probe-bind-address=:8081
+        - --metrics-bind-address=127.0.0.1:8080
+        - --leader-elect
+        command:
+        - /manager
+        image: ghcr.io/mirantiscontainers/boundless-operator:latest
```
Comment (sakshisharma84) [inline deploy/static/boundless-operator.yaml:1114]: Just to be sure, this latest tag will now reflect [v0.3.0]

**Review by sakshisharma84 — APPROVED** [review]
LGTM

File: deploy/static/boundless-operator.yaml (hunk 1 above)
Comment (ranyodh) [reply deploy/static/boundless-operator.yaml:1114]: Yes, it will.

**Review by tppolkow — APPROVED** [review]

**Pull request merged**